### PR TITLE
Reader: Add the subscription count to all reader track events, if available

### PR DESCRIPTION
**Pull request by blowery**
This should help us determine how many subscriptions an active user has.

### CODE REVIEW

File: client/reader/stats.js [hunk 1]
```diff
@@ -77,5 +79,9 @@ export function recordUnfollow( url ) {
 }
 
 export function recordTrack( eventName, eventProperties ) {
+	const subCount = SubscriptionStore.getTotalSubscriptions();
```
Comment (bluefuton) [inline client/reader/stats.js]: One potential risk is that, in #3418, we switch to incrementing `totalSubscriptions` in the store and no longer use the subscription count from the first page of the response. This count mean that getTotalSubscriptions() reports 50, 100, 150.... as the subs get loaded in.

Comment (blowery) [inline client/reader/stats.js]: Hmmmm. We probably need to keep around the total number of subs. Sounds like we'll need to mod #3418.

Comment (blowery) [inline client/reader/stats.js]: think we can land this and fixup #3418?

Comment (bluefuton) [inline client/reader/stats.js]: :+1: I've added a note to #3418.

**Comment by bluefuton** [comment]
:ship:

**Comment by bluefuton** [comment]
Events to check:

- [ ] calypso_reader_infinite_scroll_performed
- [ ] calypso_reader_blog_preview
- [ ] calypso_reader_tag_loaded
- [ ] calypso_reader_list_loaded
- [ ] calypso_reader_discover_viewed
- [ ] calypso_reader_unblock_site
- [ ] calypso_reader_site_followed
- [ ] calypso_reader_site_unfollowed

**Pull request merged**